test/Microsoft.AspNet.Mvc.FunctionalTests/ActionResultTests.cs

```diff
@@ -301,7 +301,7 @@ public class ActionResultTests
 
             // Assert
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
-            Assert.Equal("", await response.Content.ReadAsStringAsync());
+            Assert.Equal("null", await response.Content.ReadAsStringAsync());
```

Comment:
Discussed in person, the action code is specifically creating an `ObjectResult() { Value = null }`